src/legacy/server/saved_objects/service/lib/repository.js

```diff
@@ -663,6 +675,15 @@ export class SavedObjectsRepository {
     }
   }
 
+  _getIndexForType(type) {
```

Comment:
I believe `deleteByNamespace` is the only function left to take advantage of this, the other functions are great 👍